src/connectors/amazon.js

```diff
@@ -28,29 +28,27 @@ function setupPropertiesForNewPlayer() {
 
 	Connector.getTrackInfo = () => {
 		const trackContainer = document.querySelector(trackContainerSelector);
-		if (trackContainer === null) {
```

Comment:
I prefer this style of returning early as it makes the happy path less nested

Reply:
Sure, I can revert that. Is that the approach that should be taken going forward when updating/adding connectors?

Reply:
I find it often leads to cleaner and more readable code as when have multiple conditions the nesting can get excessive or the boolean logic complex.